extensions/wikia/PhalanxII/services/PhalanxService.class.php

```diff
@@ -309,4 +307,11 @@ private function sendToPhalanxDaemon( $action, $parameters ) {
 		}
 		return $res;
 	}
+
+	private function getPhalanxUrl($action) {
+		global $wgConsulUrl, $wgConsulServiceTag;
+		$baseurl = ( new Wikia\Service\Gateway\ConsulUrlProvider( $wgConsulUrl, $wgConsulServiceTag ) )->getUrl( 'phalanx' );
```

Comment:
You can just call `ConsulUrlProvider` - there's `use` statement at the top of this file.